backends/tofino/bf-p4c/CMakeLists.txt

```diff
@@ -970,7 +970,7 @@ target_include_directories(
   # FIXME: This should be pulled with FetchContent.
   PRIVATE "${BFN_P4C_SOURCE_DIR}/third_party/spdlog/include"
 )
-add_dependencies(tofinobackend ir-generated frontend genLogging bfn_p4runtime)
+add_dependencies(tofinobackend frontend genLogging bfn_p4runtime)
```

Comment:
Assume this is remove because of the frontend dependency?

Reply:
Yes. Actually, the majority `add_dependencies` are not needed at all. This seems to be an old hack used to "close" some absence of proper dependencies. This needs to be eventually removed from everywhere and replaces with proper deps (e.g. `target_link_libraries`, etc.)